integration/tests/cook/test_basic.py

```diff
@@ -1241,208 +1241,3 @@ def test_unscheduled_jobs_partial(self):
-        max_slave_cpus = max([s['resources']['cpus'] for s in slaves])
-        task_constraint_cpus = util.settings(self.cook_url)['task-constraints']['cpus']
-        # The largest job we can submit that actually fits on a slave
-        max_cpus = min(max_slave_cpus, task_constraint_cpus)
-        # The number of "big" jobs we need to submit before one will not be scheduled
-        num_big_jobs = max(1, math.floor(max_cpus / task_constraint_cpus))
-        # Use the rest of the machine, plus one half CPU so one of the large jobs won't fit
-        canary_cpus = max_slave_cpus - (num_big_jobs * max_cpus) + 0.5
-        group = {'uuid': str(uuid.uuid4()),
-                 'host-placement': {'type': 'attribute-equals',
-                                    'parameters': {'attribute': 'HOSTNAME'}}}
-        # First job - a canary job with high priority which uses canary_cpus cpus
-        canary = util.minimal_job(group=group['uuid'],
-                                  priority=100,
-                                  cpus=canary_cpus,
-                                  command='sleep 600')
-        # Second job - a big job with low priority using max_cpus cpus
-        # Based on the priority, canary should be scheduled ahead of big_job, but because
-        # of the host-placement constraint they have to be scheduled on the same host.
-        # The canary should be able to be scheduled, but big_job should never be scheduled.
-        jobs = [util.minimal_job(group=group['uuid'],
-                                 priority=1,
-                                 cpus=max_cpus)
-                for _ in range(num_big_jobs)]
-        jobs.append(canary)
-        uuids, resp = util.submit_jobs(self.cook_url, jobs, groups=[group])
-        self.assertEqual(201, resp.status_code, resp.content)
-        try:
-            util.wait_for_job(self.cook_url, canary['uuid'], 'running')
-
-            def query():
-                unscheduled_jobs, _ = util.unscheduled_jobs(self.cook_url, *[j['uuid'] for j in jobs])
-                self.logger.info(f"unscheduled_jobs response: {unscheduled_jobs}")
-                no_hosts = [reason for job in unscheduled_jobs for reason in job['reasons']
-                            if reason['reason'] == "The job couldn't be placed on any available hosts."]
-                for no_hosts_reason in no_hosts:
-                    for sub_reason in no_hosts_reason['data']['reasons']:
-                        if sub_reason['reason'] == "Host had a different attribute than other jobs in the group.":
-                            return sub_reason
-                return None
-
-            reason = util.wait_until(query, lambda r: r is not None)
-            self.assertEqual(reason['reason'],
-                             "Host had a different attribute than other jobs in the group.")
-        finally:
-            util.kill_jobs(self.cook_url, uuids)
-
-    def test_retrieve_jobs_with_deprecated_api(self):
```

Comment:
I'm guessing deleting this test was unintentional?

Reply:
PR incoming